flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/CEPITCase.java

```diff
@@ -361,17 +351,18 @@ public String select(Map<String, Event> pattern) {
 			}
 		);
 
-		result.writeAsText(resultPath, FileSystem.WriteMode.OVERWRITE);
+		result.addSink(resultSink);
```

Comment:
I didn't find any of use`DataStreamUtils#collect()` in Flink tests.
`DataStreamUtils#collect()` places in `<artifactId>flink-contrib</artifactId>` module. Are you sure that is a good idea to depend on it?

I moved classes because I saw at least four implementations String\List sinks for testing purpose. I guess it's a good idea to migrate to one of them and get rid of usage temp files where possible.

What do you think? I'm newbie in a Flink and maybe wrong understand smth